js/lib/ui.js

```diff
@@ -418,11 +418,23 @@ elgg.ui.initAccessInputs = function () {
 		}
 		var $select = $(this),
 			acl = $select.data('group-acl'),
-			$note = $('.elgg-input-access-membersonly', this.parentNode);
+			$note = $('.elgg-input-access-membersonly', this.parentNode),
+			$commentCount = $select.data('comment-count'),
```

Comment:
it's HTML5 spec: http://www.w3.org/html/wg/drafts/html/master/dom.html#custom-data-attribute
See ```element.dataset```. In DOM, ```data-foo-bar``` is accessed via ```element.dataset.fooBar```